### ISSUE TITLE
Question about Spectral loss configuration 

### ISSUE THREAD

**Issue by pjh4993** [steffen-jung/SpectralGAN-StyleGAN2#2]
Hi. Thanks for providing your nice work. I'm so glad that it is prepared for easy usage. 
I have a slight question about your spectral loss implementation.
Below is part of code network_spectral.py
```
#----------------------------------------------------------------------------
# Discriminator.

def D_spectral(
    images_in,                          # First input: Images [minibatch, channel, height, width].
    labels_in,                          # Second input: Conditioning labels [minibatch, label_size].
    
    num_channels        = 3,            # Number of input color channels. Overridden based on dataset.
    resolution          = 1024,         # Input resolution. Overridden based on dataset.
    dtype               = 'float32',    # Data type to use for intermediate activations and outputs.
    score_size          = 1,            # auxrot: Number of scalars to output. Can vary between evaluations.
    score_max           = 1,            # auxrot: Maximum number of scalars to output. Must be set at construction time.

    **_kwargs,                          # Ignore unrecognized keyword args.
):
    spectral_loss = SpectralLoss(rows=1024, cols=1024)
    
    # Inputs.
    images_in.set_shape([None, num_channels, resolution, resolution])
    images_in = tf.cast(images_in, dtype)
    
    labels_in.set_shape([None, 0])
    labels_in = tf.cast(labels_in, dtype)
    
    x = spectral_loss.spectral_vector(images_in)

    # Output layer (always trainable).
    with tf.variable_scope('Output_Spectral'):
        x = apply_bias_act(dense_layer(x, fmaps=score_max))
    scores_out = x[:, :score_size]

    # Output.
    assert scores_out.dtype == tf.as_dtype(dtype)
    scores_out = tf.identity(scores_out, name='scores_out')
    return scores_out

#----------------------------------------------------------------------------
```

Here your spectral loss has defined on 1024x1024 spatial dimension.
Should I configure it according to image resolution? Or is it intended to do much smoother Fourier transform?

I would appreciate if you let me know how you change this part when generating smaller resolution images.
